src/lib/GlobalStyles.js

```diff
@@ -102,6 +101,32 @@ export const BaseCSSReset = css`
 
 const GlobalStyle = createGlobalStyle`
   ${BaseCSSReset};
+  :root {
+    --warning-color: #e20606;
+  }
+
+  input[type=text] {
+    display: block;
+  }
+
+  button {
+    border: 2px solid gray;
```

Comment:
The color for the border and font should be #414CB3 instead

<img width="222" alt="Screen Shot 2021-07-22 at 8 22 27 PM" src="https://user-images.githubusercontent.com/9326234/126727534-ddb8c631-d268-4779-8420-96d8e9836b09.png">